packages/firestore/src/local/lru_garbage_collector_impl.ts

```diff
@@ -110,18 +110,18 @@ class RollingSequenceNumberBuffer {
  * This class is responsible for the scheduling of LRU garbage collection. It handles checking
  * whether or not GC is enabled, as well as which delay to use before the next run.
  */
-export class LruScheduler implements GarbageCollectionScheduler {
-  private hasRun: boolean = false;
+export class LruScheduler implements Scheduler {
   private gcTask: DelayedOperation<void> | null;
 
   constructor(
     private readonly garbageCollector: LruGarbageCollector,
-    private readonly asyncQueue: AsyncQueue
+    private readonly asyncQueue: AsyncQueue,
+    private readonly localStore: LocalStore
```

Comment:
I'm also curious why LocalStore became a member variable. Is it diverging from the other SDKs?